apm-agent-plugins/apm-opentracing-plugin/src/main/java/co/elastic/apm/agent/opentracing/impl/SpanContextInstrumentation.java

```diff
@@ -39,26 +41,88 @@
 import static net.bytebuddy.matcher.ElementMatchers.named;
 
 public class SpanContextInstrumentation extends OpenTracingBridgeInstrumentation {
+
+    private static final Logger logger = LoggerFactory.getLogger(SpanContextInstrumentation.class);
```

Comment:
I think this has to be public, so that the inlined advices can access this variable. Also annotate with `@VisibleForAdvice`